### PR TITLE
Fowler Brothers translation

### PR DESCRIPTION
**Pull request by gregorycrane**
https://github.com/PerseusDL/canonical-greekLit/issues/1489

### CODE REVIEW

File: data/tlg0062/tlg003/tlg0062.tlg003.perseus-eng4.xml [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Dionysus, an Introductory Lecture</title>
+<sponsor>Tufts University</sponsor>
```
Comment (AlisonBabeu) [inline data/tlg0062/tlg003/tlg0062.tlg003.perseus-eng4.xml:8]: We would normally also include the author and editor/translator name up here as wel.

**Comment by AlisonBabeu** [comment]
hi @gregorycrane from a quick look at this pull request you need to update all the cts_.xml files so that they include this new translation that you had added.

File: data/tlg0062/tlg070/tlg0062.tlg070.perseus-eng4.xml [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+ <refsDecl n="CTS">
+  <cRefPattern matchPattern="(\w+)"
+   replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+   <p>This pointer pattern extracts section.</p>
+  </cRefPattern>
+ </refsDecl>
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="en">English</language>
```
Comment (AlisonBabeu) [inline data/tlg0062/tlg070/tlg0062.tlg070.perseus-eng4.xml:68]: This should be"eng"

File: data/tlg0062/tlg070/tlg0062.tlg070.perseus-eng4.xml [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+
+<p><label>Lycinus</label> The first depends on yourself, the other on some one else.</p>
+
+<p><label>Purist</label> Quite right.</p>
+
+<p><label>Lycinus</label> Oh, you will pick up all sorts of information, if you give up thinking you know more than you do.</p>
+
+<p><label>Purist</label> I give it up from this moment.</p>
+
+<p><label>Lycinus</label> Then we will break off for the present, and take the rest another time H. and F.
```
Comment (AlisonBabeu) [inline data/tlg0062/tlg070/tlg0062.tlg070.perseus-eng4.xml:483]: I noticed that most of the translations have H. or F. or H. and F. at the end, I imagine to indicate which or both of the brothers that translated the work,but I was wondering should that be included?

File: data/tlg0062/tlg003/tlg0062.tlg003.perseus-eng4.xml (hunk 1 above)
Comment (gregorycrane) [reply data/tlg0062/tlg003/tlg0062.tlg003.perseus-eng4.xml:8]: fixed

File: data/tlg0062/tlg070/tlg0062.tlg070.perseus-eng4.xml (hunk 2 above)
Comment (gregorycrane) [reply data/tlg0062/tlg070/tlg0062.tlg070.perseus-eng4.xml:68]: fixed

**Comment by gregorycrane** [comment]
Fixed Alison's suggestions. On to the next batch!

**Pull request merged**